test/vcr_cassettes/v2_expired_token.yml

```diff
@@ -84,50 +84,5 @@ http_interactions:
         Jay)"]}],"timdex_record_id":"mit:archivesspace:VC.0002","title":"Charles J.
         Connick Stained Glass Foundation Collection"}}]},"aggregations":{"languages":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[{"key":"english","doc_count":1}]},"content_type":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"collections":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"subjects":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"content_format":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"literary_form":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"source":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[]},"contributors":{"doc_count":1,"contributor_names":{"doc_count_error_upper_bound":0,"sum_other_doc_count":0,"buckets":[{"key":"connick,
         charles j. (charles jay)","doc_count":1}]}}}}'
-  recorded_at: Thu, 07 Apr 2022 14:00:00 GMT
```

Comment:
Yeah so... this is currently dumb but all of the cassettes need these two calls in them. If you run the tests locally enough times, you'll see sporadic failures on each one. What seems to be happening is during initialization of the OpenSearch client, it makes a check for compatibility. This happens on the first test that uses the client and then the rest don't do it. For stability reasons, we need to allow that call in each cassette. Alternatively, we could figure out how to disable that Client check.

I approached this when I was making cassettes by running each test individually until it generated with both HTTP calls in the cassette and then moving on to the next one. It's not really sustainable to do that so at some point we need a better solution but for now that should work I think since we still have a smallish number of cassettes affected by this.